app/assets/javascripts/plannings.js

```diff
@@ -99,18 +99,17 @@ const spreadsheetModalExport = function(columns, planningId) {
     $('[name=spreadsheet-route]').val('');
   });
 
-  if (localStorage.spreadsheetStops) {
+  if (columns_preferences && columns_preferences['additional_stops']) {
     $.each($('.spreadsheet-stops'), function(i, cb) {
-      $(cb).prop('checked', localStorage.spreadsheetStops.split('|').indexOf($(cb).val()) >= 0);
+      $(cb).prop('checked', columns_preferences['additional_stops'].indexOf($(cb).val()) >= 0);
     });
   }
   $('.columns-export-list').sortable({
     connectWith: '#spreadsheet-columns .ui-sortable'
   });
-  var columnsExport = columns;
-  var columnsSkip = localStorage.spreadsheetColumnsSkip && localStorage.spreadsheetColumnsSkip.split('|');
-  if (localStorage.spreadsheetColumnsExport) {
-    columnsExport = localStorage.spreadsheetColumnsExport.split('|');
+  var columnsExport = (columns_preferences && columns_preferences['keep']) || [];
```

Comment:
```suggestion
  var columnsExport = (columns_preferences && columns_preferences['export']) || [];
```
To be consistent with the pre-existing feature and the current implementation. I would prefer `export` instead of `keep`